lib/zendesk_api/middleware/response/sanitise_response.rb

```diff
@@ -0,0 +1,13 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitiseResponse < Faraday::Response::Middleware
```

Comment:
I think we should spell this `SanitizeResponse`

According to http://grammarist.com/spelling/sanitise-sanitize/, while "sanitise" has become popular outside of North America, it is not even the traditional British spelling and is not recognized by the Oxford English Dictionary.